staging/src/github.com/openshift/sdn/pkg/network/node/healthcheck.go

```diff
@@ -19,18 +19,30 @@ const (
 	ovsDialDefaultAddress  = "/var/run/openvswitch/db.sock"
 )
 
+// dialAndPing connects to OVS once and pings the server. It returns
+// the dial error (if any) or the ping error (if any), or neither.
+func dialAndPing(network, addr string) (error, error) {
```

Comment:
None of the callers actually cares about the difference between the two errors...

Reply:
They don't care, but they log different things based on the output.

Reply:
yeah, but I meant, the differences aren't important, and/or you could wrap the errors with different messages here instead of logging different things there.